### ISSUE TITLE
Tensorflow exception triggered while loading model.

### ISSUE THREAD

**Issue by kraghuvamsee** [dotnet/machinelearning-modelbuilder#1189]
**System Information (please complete the following information):**
 - Model Builder Version: 16.3.0.2056001
 - Visual Studio Version: 16.8.4 Microsoft Visual Studio Community 2019

**Describe the bug**
- On which step of the process did you run into an issue: When training the model.
- Clear description of the problem: Error: Tensorflow exception triggered while loading model. System.DllNotFoundException: Unable to load DLL 'tensorflow': The specified module could not be found. (Exception from HRESULT: 0x8007007E)

**To Reproduce**
1. Create a new ConsoleApp.
2. Add Machine Learning.
3. Select Image classification. Select folder with images.
4. Train the model. Error is raised.

**Expected behavior**
Model should be trained. FYI... Text classification is working.

**Screenshots**
Please check screenshots here.
[https://docs.google.com/document/d/1PPKGgEldTQZRlvsghPqQ0hJbjVS7tl443Wj3womYBaU/edit?usp=sharing](https://docs.google.com/document/d/1PPKGgEldTQZRlvsghPqQ0hJbjVS7tl443Wj3womYBaU/edit?usp=sharing)

**Additional context**
I have tried older versions of Model Builder but did not work. Kindly look into this error.

**Comment by LittleLittleCloud** [comment]
Hi @kraghuvamsee, which CPU are you using to run image training?

**Comment by kraghuvamsee** [comment]
Hi Xiaoyun Zhang,

Thank you for taking up this issue.
My laptop has Intel i7-4510U CPU. It does support Advanced Vector Extensions (acording to https://www.cpu-world.com/).
I am glad to inform that, I am able to use Image Classification using ML.NET Model Builder, now.
 1. I have installed python3.8 and installed tensorflow.
 2. Tested tensorflow with:- python -c "import tensorflow as tf;print(tf.reduce_sum(tf.random.normal([1000, 1000])))"
 3. Tensforflow was working with python.
 4. Tried Image Classification using ML.NET Model Builder. It started working.
